src/WebUI/Pods/PodsTable.tsx

```diff
@@ -93,6 +86,18 @@ export class PodsTable extends React.Component<IPodsTableProperties> {
         return null;
     }
 
+    private _prepareSubTextData(filteredPods: V1Pod[]): void {
+        this._statusCount = {};
```

Comment:
as we were not setting this value, so we keep increasing the count of available PODs in associated pods. so had to reset for every render.